pkgs/misc/vim-plugins/generated.nix

```diff
@@ -605,12 +605,12 @@ let
 
   coc-rust-analyzer = buildVimPluginFrom2Nix {
     pname = "coc-rust-analyzer";
-    version = "2020-05-27";
+    version = "2020-06-04";
     src = fetchFromGitHub {
       owner = "fannheyward";
       repo = "coc-rust-analyzer";
-      rev = "95fe45bb82f88a5da5461a2925c7daac1ff24155";
-      sha256 = "1zczdz07fmcd7i05bz99m3a90v1dnbf4mjbh1iha0h2kyj0fj0sd";
+      rev = "acd5e7fca38dbc8ad8bfe9f187f1f4e6ee64ea88";
+      sha256 = "14qyszmyzykibdkdv38cypc8gmhaz0301prirjbpf2gijryk922b";
```

Comment:
I think there was a slight error with some of the git history with this bump,  the rev and sha256 are updated in the spirv addition.